tools/analyzer_plugin/lib/src/diagnostic/exhaustive_deps.dart

```diff
@@ -1605,12 +1605,6 @@ Iterable<Union<Identifier, NamedType>> resolvedReferencesWithin(AstNode node) sy
   }
 }
 
-extension on NamedType {
```

Comment:
These analyzer_plugin changes fix the failing analysis on Dart 3 in CI

Reply:
Should we bump the SDK lower bound to `2.19.0` in the analyzer plugin pubspec?

Reply:
Yep I can do that!